internal/engine/upper_test.go

```diff
@@ -1918,7 +1918,7 @@ func (f fakeTimerMaker) maker() timerMaker {
 		default:
 			// if you hit this, someone (you!?) might have added a new timer with a new duration, and you probably
 			// want to add a case above
-			f.t.Error("makeTimer called on unsupported duration")
+			panic("makeTimer called on unsupported duration")
```

Comment:
why?

Reply:
uhh what i mean is: any particular reason to change this?

Reply:
oops, that didn't belong in this PR

(I made this change while debugging, and remembered having issues with calls on `testing.T` not working from goroutines, but I think that's only calls to `Fatal` that are problematic)